test/TesterInternal/StreamingTests/StreamLimitTests.cs

```diff
@@ -47,12 +35,27 @@ public class StreamLimitTests : HostedTestClusterPerTest
         private string StreamNamespace;
         private readonly ITestOutputHelper output;
 
-        public override TestingSiloHost CreateSiloHost()
+        public override TestCluster CreateTestCluster()
         {
-            //MaxConsumersPerStream = 509; // ~= 64 * 1024 / 128
-            return new TestingSiloHost(siloOptions, clientOptions);
+            var options = new TestClusterOptions(initialSilosCount: 4);
```

Comment:
same as previous comment, not sure this should be 4, just the default (which is 2)